uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/projectexplorer/CamelProject.java

```diff
@@ -107,7 +107,8 @@ public void runCamelContext() {
 		}
 		AbstractWait.sleep(TimePeriod.DEFAULT);
 		try {
-			new WaitUntil(new ConsoleHasText("started and consuming from"), TimePeriod.getCustom(1200));
+			new WaitUntil(new ConsoleHasText("Starting Camel ..."), TimePeriod.getCustom(1200));
```

Comment:
it looks like this change affected other UI tests (e.g. `ProjectLocalRunTest`) which started to fail.. need to investigate more